src/datachain/lib/dc.py

```diff
@@ -723,6 +726,16 @@ def datasets(
             )
         ]
 
+        if studio and Config().read().get("studio", {}).get("token"):
```

Comment:
I think we should be throwing an exception if `studio` is specified and there is no token and team set.

Reply:
It does now.